Content.Server/GameObjects/Components/GUI/StrippableComponent.cs

```diff
@@ -0,0 +1,371 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using Content.Server.GameObjects.Components.Items.Storage;
+using Content.Server.GameObjects.EntitySystems.DoAfter;
+using Content.Server.Interfaces;
+using Content.Shared.GameObjects.Components.GUI;
+using Content.Shared.GameObjects.Components.Inventory;
+using Content.Shared.GameObjects.EntitySystems;
+using Content.Shared.Interfaces.GameObjects.Components;
+using Robust.Server.GameObjects.Components.UserInterface;
+using Robust.Server.Interfaces.GameObjects;
+using Robust.Server.Interfaces.Player;
+using Robust.Shared.GameObjects;
+using Robust.Shared.GameObjects.Systems;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Localization;
+using Robust.Shared.Log;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Server.GameObjects.Components.GUI
+{
+    [RegisterComponent]
+    public sealed class StrippableComponent : SharedStrippableComponent, IDragDrop
+    {
+        [Dependency] private IServerNotifyManager _notifyManager = default!;
```

Comment:
Dependencies don't need to be changed by user code.

```suggestion
        [Dependency] private readonly IServerNotifyManager _notifyManager = default!;
```